src/pyedb/dotnet/edb.py

```diff
@@ -1527,12 +1527,11 @@ def import_gds_file(
             ``True`` when successful, ``False`` when failed.
 
         """
-        if tech_file or map_file:
-            control_file_temp = os.path.join(tempfile.gettempdir(), os.path.split(inputGDS)[-1][:-3] + "xml")
-            ControlFile(xml_input=control_file, tecnhology=tech_file, layer_map=map_file).write_xml(control_file_temp)
-        elif tech_file:
+        if is_linux and tech_file:
```

Comment:
```suggestion
        if not is_linux and tech_file:
```
It's the opposite, we only want to fail on non linux OS.